From a3898f5d0c63fbf93b8c5b9dee6dc481a489e49d Mon Sep 17 00:00:00 2001
From: Clement Calmels <clement.calmels@intel.com>
Date: Wed, 20 Mar 2013 09:48:34 +0100
Subject: [PORT FROM MAIN] Increase number of available properties

BZ: 80168

We have reached the maximum number of available properties:
PA_COUNT_START. Currently this number is set to 247. This patch
increases this value to 496. The resulting size of the property area
will be 64k (PA_SIZE) instead of 32k.

Note: this values had already been changed on different concurrent
phones.

Change-Id: I734c849b3bf9df538e8043c2d2af753aaa56e2ee
Orig-Change-Id: I093348092b7d8ce01537942c85a14c1a4b3a10e9
Signed-off-by: Clement Calmels <clement.calmels@intel.com>
---
 init/property_service.c |   10 +++++-----
 1 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/init/property_service.c b/init/property_service.c
index cae2aee..32b77ac 100755
--- a/init/property_service.c
+++ b/init/property_service.c
@@ -153,12 +153,12 @@ out:
     return -1;
 }
 
-/* (8 header words + 372 toc words) = 1520 bytes */
-/* 1536 bytes header and toc + 372 prop_infos @ 128 bytes = 49152 bytes */
+/* (8 header words + 496 toc words) = 2016 bytes */
+/* 2048 bytes header and toc + 496 prop_infos @ 128 bytes = 65536 bytes */
 
-#define PA_COUNT_MAX  372
-#define PA_INFO_START 1536
-#define PA_SIZE       49152
+#define PA_COUNT_MAX  496
+#define PA_INFO_START 2048
+#define PA_SIZE       65536
 
 static workspace pa_workspace;
 static prop_info *pa_info_array;
-- 
1.7.4.1

